service/storage_service.cc

```diff
@@ -5979,6 +5979,7 @@ future<> storage_service::stream_tablet(locator::global_tablet_id tablet) {
     auto leaving_replica_ip = host2ip(leaving_replica.host);
 
     if (_tablet_streaming[tablet]) {
+        tm = nullptr;
```

Comment:
Please add a comment. Also expand the message. Which barrier is blocked?

Won't the streaming process also hold token metadata?

Reply:
Streaming is taken care of in https://github.com/scylladb/scylladb/issues/14146